include/picongpu/plugins/misc/rangeToWindow.hpp

```diff
@@ -0,0 +1,115 @@
+/* Copyright 2017-2022 Rene Widera
```

Comment:
Why does this file name start with a small `r` and not `R`?

Reply:
The year is wrong but whats wrong with the name? it is *R*ene

Reply:
Because it contains only a function and no class. All files which define only functions starts with lower case letters.

Reply:
Ah, i forgot that was our convention.